extra/meson.build

```diff
@@ -0,0 +1,27 @@
+desktop = 'linthesia.desktop'
+desktop_config = configuration_data()
+desktop_config.set('gettext', project_name)
+i18n.merge_file(
+  desktop,
+  type: 'desktop',
+  input: 'linthesia.desktop',
+  output: '@0@.desktop'.format(project_name),
+  po_dir: po_dir,
+  install: true,
+  install_dir: join_paths(datadir, 'applications'),
+)
+
+metainfo = 'linthesia.metainfo.xml'
+metainfo_config = configuration_data()
+metainfo_config.set('gettext', project_name)
+i18n.merge_file(
+  metainfo,
+  type: 'xml',
+  input: 'linthesia.metainfo.xml',
+  output: '@0@.metainfo.xml'.format(project_name),
+  po_dir: po_dir,
+  install: true,
+  install_dir: join_paths(datadir, 'metainfo'),
+)
+
+install_data('linthesia.xpm', install_dir: datadir / project_name / 'icons')
```

Comment:
Curious why you use `join_paths()` sometimes, and `/` string overload sugar sometimes.

Reply:
copy pasta, will fix it